README.md

```diff
@@ -103,5 +103,13 @@ In this case, the result would be:
 }
 ```
 
+## Known Issues
+
+- (Version 3.0.0) The `Json.Decode.Pipeline` lib exposes a function named `required` which is also exposed by the Elm `Html` library. To avoid compilation errors, consider aliasing the library import as follows:
+
+```elm
+import Json.Decode.Pipeline as JsonDecodePipeline exposing (decode, required, optional, hardcoded)
```

Comment:
this alternative aliases the library but not `required`. e.g.

```elm
import Json.Decode.Pipeline as VeryNamespacedPipeline exposing (required)
import HtmlAttributes as EvenMoreNamespacedAttributes exposing (required)
```
I would still have a naming collision.

Some workarounds are
- not expose multiple functions sharing the same name
  - avoiding `exposing (..)` helps with that
- calling the function _including_ the namespace, e.g. `Html.Attributes.required` and `Json.Decode.Pipeline.required` will never collide

these are Elm language concerns, which is, IMO, outside of the scope of this library's readme.